src/integrationTest/java/uk/gov/hmcts/reform/bulkscanprocessor/entity/EnvelopeRepositoryTest.java

```diff
@@ -101,6 +101,27 @@ public void findByZipFileName_should_find_envelops_in_db() {
         assertThat(resultForX).hasSize(0);
     }
 
+    @Test
+    public void findByCcdId_should_find_envelops_in_db() {
+        // given
+        Envelope e1 = envelope("A.zip", "X", Status.CREATED);
+        Envelope e2 = envelope("A.zip", "Y", UPLOAD_FAILURE);
+        Envelope e3 = envelope("B.zip", "Z", UPLOAD_FAILURE);
+        e1.setCcdId("1111123");
+        e2.setCcdId("1111123");
+        e3.setCcdId("3546684");
+        dbHas(e1, e2, e3);
+
+        // when
+        List<Envelope> resultA = repo.findByCcdId("1111123");
+        List<Envelope> resultB = repo.findByCcdId("3546684");
+        List<Envelope> resultC = repo.findByCcdId("786473");
```

Comment:
normally this should be 3 different test case

Reply:
done